ci-operator/step-registry/ipi/conf/ipi-conf-commands.sh

```diff
@@ -42,3 +42,10 @@ capabilities:
   baselineCapabilitySet: ${BASELINE_CAPABILITY_SET}
 EOF
 fi
+
+if [ X"${PUBLISH}" == X"Internal" ]; then
```

Comment:
I think here we should check `[ -n "${PUBLISH}" ]` to check if it empty or not, if not empty, set it to `publish`.